controllers/profiles/object_creators.go

```diff
@@ -202,3 +208,54 @@ func ensureWorkflowSpecConfigMapMutator(workflow *operatorapi.KogitoServerlessWo
 		}
 	}
 }
+
+// workflowPropsConfigMapCreator creates a ConfigMap to hold the external application properties
+func workflowPropsConfigMapCreator(workflow *operatorapi.KogitoServerlessWorkflow) (client.Object, error) {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      getWorkflowPropertiesConfigMapName(workflow),
+			Namespace: workflow.Namespace,
+			Labels:    labels(workflow),
+		},
+		// we could use utils.NewJavaProperties, but this way is faster
+		Data: map[string]string{
+			quarkusConfigFileName: "# Immutable Workflow Properties \n" +
+				"quarkus.http.port=" + strconv.Itoa(defaultHTTPWorkflowPort) + "\n" +
+				"quarkus.http.host=0.0.0.0\n",
```

Comment:
as the operator has access to the pod's ip, isn't better default to it?

Reply:
no, since this IP can change, we might have more pods in the same workflow, etc. It's a lot of effort to a low value.